packages/components/inline-message/src/inline-message.ts

```diff
@@ -0,0 +1,129 @@
+import type { TemplateResult } from 'lit-html';
+import type { CSSResultGroup } from 'lit';
+import type { ScopedElementsMap } from '@open-wc/scoped-elements/lit-element.js';
+import { ScopedElementsMixin } from '@open-wc/scoped-elements/lit-element.js';
+import { Icon } from '@sl-design-system/icon';
+import { Button } from '@sl-design-system/button';
+import { breakpoints } from '@sl-design-system/shared';
+import { localized, msg } from '@lit/localize';
+import { faOctagonExclamation } from '@fortawesome/pro-solid-svg-icons';
+import { LitElement, html, nothing } from 'lit';
+import { property, query } from 'lit/decorators.js';
+import styles from './inline-message.scss.js';
+
+Icon.registerIcon(faOctagonExclamation);
```

Comment:
Should this not be part of the theme?

Reply:
Icon value comes from the token, but I'm not sure if we can use it in a different way than now?
<img width="959" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/7686ff6d-d82b-4b06-9ca6-a49e78c4e73f">

Reply:
No, what I meant is that icons we use inside the SLDS web components should be defined in the theme. That way the icon is theme specific.

Reply:
So I guess this is a question to @arecuenco-dsgn and @RoaldBoerema - the icons would be the same for all themes or different per theme?

Reply:
Also a question for Diana I think. I think by design all icons used in the web components should be defined in the theme.

Reply:
Unnecessary icon registration is removed :)